modules/shared/components/atoms/Button/Button.tsx

```diff
@@ -36,27 +40,28 @@ const Button: FC<IButton.IProps> = ({
 
   return (
     <button
-      type="button"
+      /* eslint-disable react/button-has-type */
+      type={buttonType}
       onClick={onClick}
       className={btnClasses}
       disabled={disabled}
     >
       {leftIcon && (
         <span>
-          <ButtonArrowDown />
+          <ArowDown />
```

Comment:
I know it was my suggestion 😂😂 to name it `<ArrowDown />` but the more explicit the better. 
`<ArrowDownIcon />` is better. or you can suggest a better one.